.github/workflows/pr-contract-check.yml

```
@@ -0,0 +1,166 @@
+    strategy:
+      matrix:
+        contract: ${{ fromJson(needs.detect-changes.outputs.contracts) }}
+    
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0  # Need full history
+      
+      - name: Set up Rust
+        uses: dtolnay/rust-toolchain@stable
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)